cmd/limactl/main.go

```diff
@@ -76,6 +76,7 @@ func newApp() *cobra.Command {
 		newListCommand(),
 		newDeleteCommand(),
 		newValidateCommand(),
+		newConfigCommand(),
```

Comment:
Maybe under a `debug` subcommand

https://github.com/lima-vm/lima/issues/443#issuecomment-985221726

Reply:
I don't think it is related to debugging lima, so maybe just leave it out then